Airports/EGPC/VRPs.txt

```diff
@@ -3,4 +3,4 @@
 *Keiss Village:N058.32.00.000:W003.07.24.000
 *Loch Watten:N058.29.00.000:W003.20.06.000
 *Lybster Village:N058.18.00.000:W003.17.06.000
-*Thrumster Masts:N058.23.37.907:W003.07.27.528
\ No newline at end of file
+*Thrumster Masts:N058.23.37.900:W003.07.27.520
```

Comment:
why is this being changed?
```suggestion
*Thrumster Masts:N058.23.37.907:W003.07.27.528
```
![image](https://github.com/VATSIM-UK/UK-Sector-File/assets/61326713/03a0e3f1-e315-4b70-a37d-6ec145ddf83f)

Reply:
An excellent question!
Also I should probably have downloaded that dynamic VRP list, it looks a lot easier to work with than the CSV one haha